docs/deprecations.md

```diff
@@ -6,6 +6,13 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+## Http Api provider tags
```

Comment:
Sorry - now I see that I've missed one thing here. This should be listed below the `How to disable specific deprecation logs`. It should also include link similar to `<a name="MISSING_COMMANDS_OR_OPTIONS_AT_CONSTRUCTION"><div>&nbsp;</div></a>` but with proper deprecation code - please take a look at the other deprecation sections on how there are constructed. 

Sorry for missing that during previous reviews

Reply:
just pushed, thanks for the detailed comment and sorry for my wording

Reply:
No problem @jayasai470, thank you :bow: There's one thing that I've mentioned which I believe was not addressed - moving this deprecation below `How to disable specific deprecation logs` - that section should be the first one listed and specific deprecations as the one added should be listed right below it